### ISSUE TITLE
Text fields for Notes and Background Description, Relationships, and Goals do not save text.  

### ISSUE THREAD

**Issue by jamakkon** [Foxfyre/expanse#27]
**Describe the bug**
Filling in information for the text field under Notes tab has no impact. The text editor functions as it should, but clicking save makes the input disappear. The applies to all three text fields under Background tab.  

**To Reproduce**
1. Create a new actor by clicking 'Create Actor' button under Actors Directory
2. Click on 'Notes' tab in the newly created character sheet.
3. Click on the 'Edit' icon at the top-right corner of the text field.
4. Type in text.
5. Click on 'Save' button in the editor toolbar.
6. See error: The editor closes, but the text field is unchanged.

**Expected behavior**
One would expect to see the edited text in the text field.

**Desktop (please complete the following information):**
 - OS: macOS 12.3.1
 - Browser: Firefox
 - Version: 99
 - The Expanse RPG (AGE): 0.8.2
 -  Foundry VTT: v9 (build 268)

**Comment by Foxfyre** [comment]
I am unable to duplicate this error. Are there any errors  showing in the console?

**Comment by jamakkon** [comment]
There are some warnings, but they don't trigger every time right on the spot. The "Expected media feature... " shows up pretty consistently.
```
injection entry point. starting message loop with extension sandbox and waiting for settings... [contentscript.js:863:13](moz-extension://77f286b6-5377-d14b-9a43-72aefcdf2291/contentscript.js)
Expected media feature name but found ‘-ms-high-contrast’. [content.min.css:7:14052](...../scripts/skins/ui/oxide/content.min.css)
Expected media feature name but found ‘-ms-high-contrast’. content.min.css:7:14079
Error: Promised response from onMessage listener went out of scope 5 [background.js:842:170](moz-extension://5b2d79d0-d9f1-204e-9b7f-bbce1188e1c8/dist/background.js)
```

**Comment by jamakkon** [comment]
Oh, for what it's worth, and the problem occurs on Safari 15.4 as well.

**Comment by soulmanagement** [comment]
I just tried it before updating my version of this world: It worked in 0.8.0. with Firefox, it does no longer work for me after updating to 0.8.2., again with Firefox.
It does also not work for the gamemaster in Foundry VTT itself, after updating to 0.8.2.

I believe in 0.8.0. we entered the editor immediately, in 0.8.2. we have to click the editor button to open the editor. Perhaps this has something to do with it?
